File: IT/Input-Output/Esercizi/Java/Esercizio17.java
/* file Esercizio17.java */

import java.util.List;
import java.io.File;
import java.io.IOException;
import java.nio.file.Paths;
import java.nio.file.Files;
import java.nio.charset.Charset;

public class Esercizio17 {
	public static void main(String [] args) {
		try {
			File doc = new File("file.txt");

			List<String> lines = Files.readAllLines(Paths.get(doc.toURI()), Charset.defaultCharset());

			int j = lines.size();

			for (int i = j - 3; i < j; i++) {
				System.out.println(lines.get(i));
			}
		}
		catch (IOException e) {
			System.out.println(e);
		}
	}
}

File: IT/Input-Output/Esercizi/Python/esercizio17.py
# file esercizio17.py

from __future__ import print_function

try:
	doc = open("file.txt", "r")
	lines = doc.readlines()

	j = len(lines)
	i = j - 3 if j >= 3 else 0

	for i in range(i, j):
		print(lines[i], end="")

	doc.close()

except IOError as e:
	print(e)

